Lib/test/test_netrc.py

```diff
@@ -1,45 +1,139 @@
 import netrc, os, unittest, sys, textwrap
-from test.support import os_helper
+
+from test import support
+from contextlib import ExitStack
```

Comment:
```suggestion
from contextlib import ExitStack
from test import support
from test.support import os_helper
```

And use `os_helper` in the file. This will reduce the diff as well.